src/Spd.Manager.Licence/BizPortalUserManager.cs

```diff
@@ -0,0 +1,50 @@
+﻿using AutoMapper;
+using MediatR;
+using Spd.Manager.Shared;
+using Spd.Resource.Repository.Org;
+using Spd.Resource.Repository.PortalUser;
+using Spd.Resource.Repository.User;
+using Spd.Utilities.Shared.Exceptions;
+using System.Net;
+
+namespace Spd.Manager.Licence;
+internal class BizPortalUserManager
+    : IRequestHandler<BizPortalUserCreateCommand, BizPortalUserResponse>,
+    IBizPortalUserManager
+{
+    private readonly IMapper _mapper;
+    private readonly IPortalUserRepository _portalUserRepository;
+    private readonly IOrgRepository _orgRepository;
+
+    public BizPortalUserManager(IMapper mapper, IPortalUserRepository portalUserRepository, IOrgRepository orgRepository)
+    {
+        _mapper = mapper;
+        _portalUserRepository = portalUserRepository;
+        _orgRepository = orgRepository;
+    }
+
+    public async Task<BizPortalUserResponse> Handle(BizPortalUserCreateCommand request, CancellationToken ct)
+    {
+        PortalUserListResp existingUsersResult = await _portalUserRepository.QueryAsync(
```

Comment:
this probably including screening portal user. So, we needs to update queryAsync, we should be able to filter out primary manager and manager

Reply:
Done.